views/js/scoring/processor/expressions/operators/roundTo.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technlogies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+/**
+ * The roundTo operator processor.
+ * @see http://www.imsglobal.org/question/qtiv2p1/imsqti_infov2p1.html#element106470
+ *
+ * @author Bertrand Chevrier <bertrand@taotesting.com>
+ */
+define([
+    'lodash',
+    'taoQtiItem/scoring/processor/expressions/preprocessor',
+    'taoQtiItem/scoring/processor/errorHandler'
+], function(_, preProcessor, errorHandler){
+    'use strict';
+
+    /**
+     * Process operands and returns roundTo result.
+     * @type {OperatorProcessor}
+     * @exports taoQtiItem/scoring/processor/expressions/operators/roundTo
+     */
+    var roundToProcessor = {
+
+        engines: {
+            significantFigures: function (value, exp) {
+                return decimalAdjust('round', value, exp);
+            },
+            decimalPlaces: function (value, exp) {
+                return decimalAdjust('floor', value, exp);
+            }
+        },
+
+        constraints : {
+            minOperand  : 1,
+            maxOperand  : 1,
+            cardinality : ['single'],
+            baseType    : ['integer', 'float']
+        },
+
+        operands   : [],
+
+        /**
+         * @returns {?ProcessingValue} a single boolean
+         */
+        process : function(){
+
+             var roundingMode = _.isFunction(this.expression.attributes.roundingMode) ? this.expression.attributes.roundingMode : roundToProcessor.engines.significantFigures;
+            var figures = _.isNumber(this.expression.attributes.figures) ? this.expression.attributes.figures : undefined;
+
```

Comment:
this.expression.attributes are directly retrieved from the QTI XML. 
`this.expression.attributes.roundingMode` will be a string and `this.expression.attributes.figures` will be either a string or number that represents a string (use parseInt) or the reference to a variable, then you have to get the variable from `this.state` and get it's value, like (but implement it more smartly):

```javascript
var figures = parseInt(this.expression.attributes.figures, 10);
if(_.isNumber(figures) && !_.isNaN(figues)){
 //...
} else {
  //try to get the variable value
  if(this.state[this.expression.attributes.figures]){
    figures = parseInt(this.state[this.expression.attributes.figures].value, 10);
       if(_.isNumber(figures) && !_.isNaN(figues)){
           //...
       }
   }
}
```